spec/lib/auth0/mixins/httpproxy_spec.rb

```diff
@@ -148,6 +169,26 @@
           .and_raise(@exception)
         expect { @instance.send(http_method, '/test') }.to raise_error(Auth0::Unauthorized)
       end
+      
+      it "should raise Auth0::RateLimitEncountered on send http #{http_method} method
```

Comment:
I'm struggling to understand what this test is supposed to be doing. I see that `headers` are different between the two. The expects are the same though. Can you explain?

Reply:
These are essentially the same, but one is in the block that tests the `GET`/`DELETE` method, and one is in the block that tests the `POST`/`PUT`/`PATCH` methods; I was simply following the pattern established by the other, existing tests.